components/sections/footer/variant_a.tsx

```diff
@@ -14,140 +19,107 @@ function VariantA({
   socialMedia,
 }: FooterProps) {
   return (
-    <section>
-      <div className="radius-for-skewed bg-gray-50 py-20">
-        <div className="container mx-auto px-4">
-          <div className="mb-5 flex flex-wrap lg:mb-20">
-            <div className="mb-5 w-full lg:w-1/5">
-              {logo?.image && (
-                <Link
-                  aria-label={
-                    logoLink(logo) === "/"
-                      ? "Go to home page"
-                      : `Go to ${logoLink(logo)}`
-                  }
-                  className="text-3xl font-bold leading-none"
-                  href={logoLink(logo)}
+    <section className="py-20 bg-gray-50">
+      <Container>
+        <Flex wrap className="mb-5 lg:mb-20">
+          <div className="w-full mb-5 lg:w-1/5">
+            {logo?.image && (
+              <Link
+                aria-label={
+                  logoLink(logo) === "/"
+                    ? "Go to home page"
+                    : `Go to ${logoLink(logo)}`
+                }
+                className="text-3xl font-bold leading-none"
+                href={logoLink(logo)}
+              >
+                <Image
+                  className="h-14"
+                  src={urlFor(logo?.image)}
+                  sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
+                  width={132}
+                  height={56}
+                  alt={logo?.alt ?? "footer-logo"}
+                />
+              </Link>
+            )}
+          </div>
+          <div className="w-full mb-5 lg:w-1/5">
+            <Text muted className="leading-loose ">
```

Comment:
Kindly conditionally render Text and Copyright elements to only display if it is defined/has value.